apps/dtsse/dtsse-dashboard-github-ingestion/aat/01.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dtsse-dashboard-github-ingestion
+spec:
+  releaseName: dtsse-dashboard-github-ingestion
+  values:
+    job:
+      schedule: "10,30,50 * * * *"
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: aat
```

Comment:
you shouldn't need any of this it will be added by a file at the environment level

Reply:
So remove: 

```
      jobKind: CronJob
      enableKeyVaults: true
      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
      environment: aat
```

Here and in the 00.yaml file?

Reply:
jobKind may need to be there

Reply:
probably keep the jobKind